components/profile/ProfileDetail.vue

```diff
@@ -483,7 +484,7 @@ const followConfig: ButtonConfig = {
       initiatorAddress: accountId.value,
       targetAddress: id.value as string,
     }).catch(() => {
-      if (isBase.value) {
+      if (isSub.value) {
```

Comment:
```suggestion
      if (isEvm.value) {
```

Reply:
It's the old commit on the changes. Could you reload this page?

Reply:
Yup I had old changes